### PR TITLE
Fix InfoWedge when no beatmap available

### PR DESCRIPTION
**Pull request by huoyaoyuan**
*hope I won't break anything*

### CODE REVIEW

File: osu.Game/Beatmaps/DummyWorkingBeatmap.cs [hunk 1]
```diff
@@ -26,7 +26,7 @@ public DummyWorkingBeatmap(OsuGameBase game)
                     Title = "no beatmaps available!",
                     Author = "no one",
                 },
-                BeatmapSet = new BeatmapSetInfo(),
+                BeatmapSet = new BeatmapSetInfo { DeletePending = true },
```
Comment (peppy) [inline osu.Game/Beatmaps/DummyWorkingBeatmap.cs]: this deserves a comment explaining why its here, at very least.

**Comment by peppy** [comment]
would help to explain what was broken int he first place (with screenshots would do fine)

**Comment by huoyaoyuan** [comment]
Before:
##### First load:
image
##### Back to main menu and load again:
image

The reason is that if the `WorkingBeatmap` object actual changes (from menu music to dummy), the info wedge will be shown. But since the second load, the `!Beatmap.IsDefault` will prevent showing it.

File: osu.Game/Beatmaps/DummyWorkingBeatmap.cs [hunk 2]
```diff
@@ -26,7 +26,7 @@ public DummyWorkingBeatmap(OsuGameBase game)
                     Title = "no beatmaps available!",
                     Author = "no one",
                 },
-                BeatmapSet = new BeatmapSetInfo(),
+                BeatmapSet = new BeatmapSetInfo { DeletePending = true }, //let song select going to status that no beatmaps available
```
Comment (smoogipooo) [inline osu.Game/Beatmaps/DummyWorkingBeatmap.cs]: How about "let song select show that no beatmaps are available if this is the only one"?

**Review by smoogipoo — CHANGES_REQUESTED** [review]

File: osu.Game/Beatmaps/DummyWorkingBeatmap.cs (hunk 2 above)
Comment (smoogipooo) [reply osu.Game/Beatmaps/DummyWorkingBeatmap.cs]: Actually this doesn't have any effect in general as far as I can tell?

Comment (huoyaoyuan) [reply osu.Game/Beatmaps/DummyWorkingBeatmap.cs]: your suggestion sounds good

**Comment by huoyaoyuan** [comment]
Another thing: the rotation transform of InfoWedge looks weird in my opinion.

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**